pulpcore/cli/ansible/context.py

```diff
@@ -69,6 +70,7 @@ class PulpAnsibleCollectionRemoteContext(PulpRemoteContext):
     HREF = "ansible_collection_remote_href"
     ID_PREFIX = "remotes_ansible_collection"
     collection_nullable = ["auth_url", "requirements_file", "token"]
+    HREF_PATTERN = r"remotes/(?P<plugin>ansible)/(?P<resource_type>collection)/"
```

Comment:
Yes, exactly. Similarly to pulp_container, we want to prevent users from using HREF that does not refer to a currently used context.
```
pulp container repository -t push show --repository /pulp/api/v3/repositories/container/container/7864a76f-5d5b-4b7b-9587-2fe076abf4f0/
Error: '/pulp/api/v3/repositories/container/container/7864a76f-5d5b-4b7b-9587-2fe076abf4f0/' is not a valid href for repository.
```
 ^ the CLI expects repositories/container/container-push/